packages/@aws-cdk/aws-ecs/test/ec2/test.ec2-task-definition.ts

```diff
@@ -128,7 +129,42 @@ export = {
 
       test.done();
     },
+    "correctly sets container dependenices"(test: Test) {
```

Comment:
Can you also create a test where you specify a different container dependency condition (COMPLETE, SUCCESS)